### DISCUSSION TITLE
Unable to run prisma with cloud cloudfare workers

### DISCUSSION THREAD

**Discussion by adityarawat45** [prisma/prisma#24726] [Q&A] [answered]
### Bug description

Error: PrismaClient is not configured to run in Cloudflare Workers.  this is what i got while testing my apis

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by jkomyno** [comment, marked as answer]
Hi @adityarawat45, are you already aware of the `driverAdapters` preview feature and of [this Prisma Docs page](https://www.prisma.io/docs/orm/prisma-client/deployment/edge/deploy-to-cloudflare)?

Here's an example of how to use [Prisma + Cloudflare Workers](https://github.com/prisma/ecosystem-tests/tree/dev/driver-adapters-wasm/d1-cf-basic).

We also recommend that you fill your issue template next time, so that we can provide more accurate answers, and e.g. be aware of which kind of database you'd like Prisma to work with in your setup. Thanks!

**Reply by adityarawat45** [reply]
Oh yah, actually i got the solution for it.
I was importing PrismaClient from @prisma/client instead of @prisma/client/edge.
My bad!
